rhel8/profiles/stig.profile

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: '[DRAFT] DISA STIG for Red Hat Enterprise Linux 8'
+
+description: |-
+    This profile contains configuration checks that align to the
+    [DRAFT] DISA STIG for Red Hat Enterprise Linux 8.
+
+    In addition to being applicable to Red Hat Enterprise Linux 8, DISA recognizes this
+    configuration baseline as applicable to the operating system tier of
+    Red Hat technologies that are based on Red Hat Enterprise Linux 8, such as:
+
+    - Red Hat Enterprise Linux Server
+    - Red Hat Enterprise Linux Workstation and Desktop
+    - Red Hat Enterprise Linux for HPC
+    - Red Hat Storage
```

Comment:
How does this profile apply to Ceph?

Reply:
e.g. by that same logic, this would apply to OpenShift.

Reply:
> How does this profile apply to Ceph?

@shawndwells because the el7 has the statement that it applies to `Red Hat Storage`, so logically, the same statement applies to el8

Reply:
> e.g. by that same logic, this would apply to OpenShift.

Partially true, that is of course if the node is pure RHEL.